### ISSUE TITLE
Добавочные статусы

### ISSUE THREAD

**Issue by TristisOris** [qt-kaneko/Shikiplayer#9]
факт наличия плеера добавляет на многие страницы подобный код:
![image](https://github.com/qt-kaneko/Shikiplayer/assets/1270117/e6611295-2507-48ec-ab01-8b2f9e7d8de0)

и в список тоже, хотя еще даже не начинал смотреть этот тайтл.
![image](https://github.com/qt-kaneko/Shikiplayer/assets/1270117/7ded57ac-4db4-452b-8d66-7857b169f504)

сначала думал это просто визуал, но при выгрузке списка в xml получается:

```
  <anime>
    <series_title>On Air Dekinai!</series_title>
    <series_type>tv</series_type>
    <series_episodes>12</series_episodes>
    <series_animedb_id>50285</series_animedb_id>
    <my_watched_episodes>12</my_watched_episodes>
    <my_times_watched>0</my_times_watched>
    <my_score>6</my_score>
    <my_status>Completed</my_status>
    <shiki_status>Completed</shiki_status>
    <my_comments>

Shikiplayer Data: {"episodeTime":0,"translation":2148}</my_comments>
    <update_on_import>1</update_on_import>
  </anime>
```

тоесть это доп поле хранящееся в базе?

**Comment by qt-kaneko** [comment]
Да. У Shikiplayer'а нет своей базы чтобы хранить ваши данные, из-за этого он сохраняет информацию в 'заметке' к аниме. Там хранится выбранная вами озвучка и секунда на которой вы закончили просмотр. Эта информация сохраняется в ходе просмотра серии, её не должно быть если вы ни разу не запускали плеер.

**Comment by TristisOris** [comment]
а. Никогда не пользовался заметками, только слышал, что они есть. Тогда нет проблем, раз ничего не сломается.
